patroni/api.py

```diff
@@ -1480,6 +1494,38 @@ def __init__(self, patroni: Patroni, config: Dict[str, Any]) -> None:
         self.reload_config(config)
         self.daemon = True
 
+    def __construct_server_tokens(self, token_config: str) -> str:
+        """Construct the value for the ``Server`` HTTP header based on *server_tokens*.
+
+        :param server_tokens: the value of ``restapi.server_tokens`` configuration option.
+
+        :returns: a string to be used as the value of ``Server`` HTTP header.
+        """
+        token = token_config.lower()
+
+        logger.debug('restapi.server_tokens is set to "%s".', token_config)
+        if token not in ('original', 'full', 'productonly', 'minimal'):
+            logger.warning('restapi.server_tokens is set to "%s". Patroni will not modify the Server header. '
+                           'Valid values are: "Full", "Minimal", "ProductOnly".', token_config)
+            return ""
+        
+        # If 'original' is set, we do not modify the Server header.
+        # This is useful for compatibility with existing setups that expect the original header.
+        if token == 'original':
+            logger.info('restapi.server_tokens is set to "Original". Patroni will not modify the Server header.')
```

Comment:
It will log it every time when reload with default config happens.

Reply:
Good point - will remove the logging.